Sources/Bond/Observable Collections/Changeset.swift

```diff
@@ -24,11 +24,17 @@
 
 import Foundation
 
+public protocol DiffProtocol {
+    var isEmpty: Bool { get }
+    var isReload: Bool { get }
```

Comment:
The `isReload` name seems a bit off to me (and we can fix this later, just opening up conversation while this PR is open). Maybe something like:

```suggestion
    var wantsReload: Bool { get }
```

Might be more accurate and descriptive? I'm not sure. I also considered `shouldReload`. @srdanrasic might have some ideas - his naming is always spot-on.